ui/apps/profile/src/components/profile-cards/profile-page-header.tsx

```diff
@@ -142,6 +142,7 @@ export const ProfilePageHeader: React.FC<ProfilePageCardProps> = props => {
         followLabel={t('Follow')}
         unfollowLabel={t('Unfollow')}
         descriptionLabel={t('About me')}
+        badgesLabel={t('Badges')}
```

Comment:
is this the badges section title? or every badge will be labeled with this prop?

Reply:
the section title, just like the descriptionLabel for the About me section